sphinx/ext/coverage.py

```diff
@@ -65,6 +66,86 @@ def _add_row(col_widths: list[int], columns: list[str], separator: str) -> Itera
     yield _add_line(col_widths, separator)
 
 
+def _load_modules(mod_name: str, ignored_module_exps: list[re.Pattern[str]]) -> set[str]:
```

Comment:
I wonder if we'd be able to use ``importlib`` (e.g. [``importlib.util.find_spec``](https://docs.python.org/dev/library/importlib.html#importlib.util.find_spec); [``importlib.util.spec_from_file_location``](https://docs.python.org/dev/library/importlib.html#importlib.util.spec_from_file_location)) to avoid importing the modules, given we only want to find their names and subpackages.

Reply:
Does it matter? We won't import modules that we don't want to document as we check the module name first, and we import all the other modules anyway via later calls to `import_module`.